system/cu/cu_main.c

```diff
@@ -342,8 +366,37 @@ int main(int argc, FAR char *argv[])
       goto errout_with_devinit;
     }
 
-  enable_crlf_conversion(g_cu.outfd);
-  set_baudrate(g_cu.outfd, baudrate, parity, rtscts);
+#ifdef CONFIG_SERIAL_TERMIOS
+  /* remember serial device termios attributes */
+
+  ret = tcgetattr(g_cu.outfd, &g_tio_dev);
+  if (ret)
+    {
+      fprintf(stderr, "cu_main: ERROR during tcgetattr(): %d\n", errno);
+      goto errout_with_outfd;
+    }
+
+  /* remember std termios attributes if it is a tty. Try to select
+   * right descriptor that is used to refer to tty
+   */
+
+  if (isatty(fileno(stderr)))
+    fd_std_tty = fileno(stderr);
+  else if (isatty(fileno(stdout)))
+    fd_std_tty = fileno(stdout);
+  else if (isatty(fileno(stdin)))
+    fd_std_tty = fileno(stdin);
+  else
+    fd_std_tty = -1;
+
+  if (fd_std_tty >= 0)
+    tcgetattr(fd_std_tty, &g_tio_std);
```

Comment:
Curly brackets are required even for one-line/empty if/for/while statements.

Reply:
Coding Standard:  https://cwiki.apache.org/confluence/display/NUTTX/Coding+Standard#ifthenelse